deploy/modules/db-onboarder/onboard.tpl

```diff
@@ -51,8 +51,19 @@ if ! curl --fail -k 'https://127.0.0.1:8443/dsf/api/v1/data-sources/${db_arn}' -
         --header "Authorization: Bearer $hub_token" \
         --header 'Content-Type: application/json' \
         --data-raw '${database_asset_data}'
-    echo ********Sleeping 30 seconds before enabling audit logs********
-    sleep 30
+fi
+
```

Comment:
Other API calls we use are also released starting 4.10. I think this condition should be part of the entire module rather than just those specific calls.

Reply:
The API is not planned to be at 4.11, they are planning to add the sync to be part of add API